default.nix

```diff
@@ -38,42 +39,65 @@ let
     };
   };
 
+  config.zfs_filesystem = q: x: {
```

Comment:
q? x?

Reply:
q is the state upwards and x the configuration downwards